test/e2e/mixedcpus_test.go

```diff
@@ -55,6 +54,21 @@ var _ = Describe("Mixedcpus", func() {
 		DeferCleanup(deleteNamespace, fixture.TestingNS)
 	})
 
+	Context("requests more devices than node has", func() {
+		It("should generate more devices", func() {
+			By("create deployment which asks more devices than the node has")
+			pod := pods.Make("pod-test", fixture.TestingNS.Name, pods.WithRequests(corev1.ResourceList{
+				deviceplugin.MutualCPUDeviceName: resource.MustParse("1"),
+			}))
+			dp := deployments.Make("dp-test", fixture.TestingNS.Name, deployments.WithPodSpec(pod.Spec), deployments.WithReplicas(50))
```

Comment:
how do we know this amount of replicas is greater than the value exposed by the node? is that an assumption or can we enforce this programmatically? (querying the node object probably)

Reply:
We can actually you're right it would be smarter.
Essentially it should be bigger than: number of nodes X  number of devices at each node

Reply:
Done. Thanks